RESOURCES.md

```diff
@@ -48,73 +56,81 @@ Below are links to various places you can find the Kentico developer community.
 
 - [Kentico Certified Developers Group on LinkedIn](https://www.linkedin.com/groups/3172023/) - Kentico LinkedIn group for all Kentico certified developers
 
+- [Kentico Xperience Product Suggestions](https://portal.productboard.com/kenticoxperience/) - Product Board site to discuss, recommend, and vote on ideas for the Kentico Xperience platform
+
 ### Kentico Company & Employees on Social Media
 
 #### Kentico on Twitter
 
-- [Kentico](https://twitter.com/Kentico) - Kentico
+- [Kentico](https://twitter.com/Kentico) - Kentico Company account
 
-- [Kentico Kontent](https://twitter.com/KenticoKontent) - Kentico Kontent
+- [Kentico Kontent](https://twitter.com/KenticoKontent) - Kentico Kontent product
 
-- [Kentico Design](https://twitter.com/KenticoDesign) - Kentico's internal designers
+- [Kentico Xperience](https://twitter.com/KenticoXP) - Kentico Xperience product
 
-- [Michal Kadák (Kentico Xperience)](https://twitter.com/MichalKadak) - Product Director - Kentico Xperience
+- [Kentico Design](https://twitter.com/KenticoDesign) - Kentico's internal designers
 
-- [Ondřej Chrastina (Kentico Xperience/Kentico Kontent)](https://twitter.com/ChrastinaOndrej) - Kentico Developer Advocate
+- [Ondřej Chrastina (Kentico Kontent)](https://twitter.com/ChrastinaOndrej) - Kentico Kontent Developer Advocate
```

Comment:
I updated the devrel product associates. Am I correct that @OnyxPrime is the only Xperience devrel person?

Reply:
You are correct. I am currently the only Xperience DevRel member